### PR TITLE
Remove lsb

### PR DESCRIPTION
**Pull request by rnelson0**
Resolves issue #41

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ghoneycutt-nfs",
-  "version": "1.8.0",
+  "version": "1.8.1",
```
Comment (ghoneycutt) [inline metadata.json]: Can you please unstage this and rebase your commit? It is good practice to not bump versions in a commit as the upstream maintainer will manage this as they might release other code before this.

**Comment by ghoneycutt** [comment]
retriggered travis. not sure why this was failing everywhere.

File: manifests/idmap.pp [hunk 2]
```diff
@@ -82,7 +82,7 @@
   case $::osfamily {
     'RedHat' : {
       $default_pipefs_directory = 'UNSET'
-      case $::lsbmajdistrelease {
+      case $::operatingsystemmajrelease {
```
Comment (ghoneycutt) [inline manifests/idmap.pp]: You'll need to update the corresponding spec tests for RedHat to use this fact.

**Comment by rnelson0** [comment]
@ghoneycutt I split this into two commits, one each for manifests and rspec test changes, let me know if you want those merged. Otherwise the rebase is ready for review.

**Comment by ghoneycutt** [comment]
Hi @rnelson0, Thanks again! Please rebase into one commit as either commit is not mergeable by itself.

I'm looking into why travis-ci is failing though. Submitted PR #52. After I get it merged, could you rebase against master and do your search/replace magic again :) Hopefully that fixes the travis-ci issue.

**Comment by ghoneycutt** [comment]
Got it merged.

**Comment by rnelson0** [comment]
@ghoneycutt Done, ready for review.

**Comment by rnelson0** [comment]
Whoops, hold off, just saw the new tests you added in that PR.

**Comment by rnelson0** [comment]
@ghoneycutt OK really good to go this time!

**Comment by ghoneycutt** [comment]
@rnelson0 great work!

**Comment by ghoneycutt** [comment]
Released in v1.9.0

**Pull request merged**